server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -763,6 +764,12 @@ protected Node(
                 fileCacheCleaner
             );
 
+            final AdmissionControlService admissionControlService = AdmissionControlService.newAdmissionControllerService(
```

Comment:
If we are planning to add node level stats , can we add AC service to 'NodeService' file as well ? This can be done as part of stats PR as well if not now.

Reply:
Yes It will be added in stats PR's.